pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter_test.go

```diff
@@ -163,12 +174,6 @@ var _ = Describe("Pod eviction admitter", func() {
 						&metav1.PatchOptions{}).
 					Return(nil, fmt.Errorf("err"))
 
-				clusterConfig := newClusterConfigWithFeatureGate(virtconfig.LiveMigrationGate)
-
-				podEvictionAdmitter := PodEvictionAdmitter{
-					ClusterConfig: clusterConfig,
-					VirtClient:    virtClient,
-				}
```

Comment:
Can you explain why these are removed?

Reply:
commit https://github.com/kubevirt/kubevirt/pull/7086/commits/eb84d1460fcf7a0befc7a451d557b91eac4f5fa4 removes it, It's duplicated code (it's done in the BeforeEach).